### PR TITLE
updated the react restriction to include react 18 - tested; works fine

### LINKED ISSUE #137: [Bug] Doesn't works with React 18
**First, I want to thank you so much for this library! I enjoy using this.**
 Unfortunately, this library does not work with React 18.

## Package.json
image

## Describe the bug
Please provide a clear and concise description of what the bug is. Include relevant operating system, browser and other environment information.

## How To Reproduce

1. Make sure you are on React 18.x.
2. Make sure you have the latest version of react-live-chat-loader (2.7.3).
3. npm install
4. See error bellow

## Expected behaviour
A clean npm install

## Screenshots
image

### PR DESCRIPTION
**Pull request by cbschuld**
## What does this PR introduce?

Lame and simple update to support react 18 - tested it locally and in production and appears 👍

- updates package.json requirements to allow for react 18
- fixes #137
- updates CHANGELOG as well just help speed up a release (hopefully that is helpful)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
     "url": "https://github.com/calibreapp/react-live-chat-loader.git"
   },
   "peerDependencies": {
-    "react": "^16.14.0 || ^17.0.0"
+    "react": "^16.14.0 || ^17.0.0 || ^18.0.0",
```
Comment (benschwarz) [inline package.json:41]: According to Vercel's build, this is invalid 

image

Comment (benschwarz) [inline package.json:41]:
```suggestion
    "react": "^16.14.0 || ^17.0.0 || ^18.0.0"
```

**Comment by benschwarz** [comment]
I think this is good to go. We'll bump the version and release it shortly.

**Comment by thefoxis** [comment]
@cbschuld no contribution is lame! 😸 thanks so much 🙏🏻

**Comment by vadimdemedes** [comment]
Hey @benschwarz, could you release a new version with this fix in it?

**Comment by benschwarz** [comment]
@cbschuld Thanks again for your contribution 💪 

@vadimdemedes This has been released as 2.7.4

**Pull request merged**